packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -8,7 +8,7 @@ import ResizeObserver from 'resize-observer-polyfill';
 import ContentContainer from 'terra-content-container';
 import { injectIntl, intlShape, FormattedMessage } from 'react-intl';
 
-import KeyCode from 'keycode-js';
+import * as KeyCode from 'keycode-js';
```

Comment:
i know keycode-js is relatively small. but it's probably best to only import what we use, reducing import size
```suggestion
import { KEY_SHIFT, KEY_TAB, KEY_TAB } from 'keycode-js';
```

Reply:
The imported code would only be minimized if it is built with tree shaking enabled in the webpack config, but we should still change this to better support that configuration.

Reply:
Sure sounds good. Removed wildcard imports for keycode-js: https://github.com/cerner/terra-clinical/pull/730/commits/7134b1ed4fa10b287f58607f9e1a251c4f6edd60

Reply:
that's good to know!